packages/marketplace/src/marketplace.gen.ts

```diff
@@ -532,11 +532,12 @@ export interface Admin {
   createCurrency(args: CreateCurrencyArgs, headers?: object, signal?: AbortSignal): Promise<CreateCurrencyReturn>
   createCurrencies(args: CreateCurrenciesArgs, headers?: object, signal?: AbortSignal): Promise<CreateCurrenciesReturn>
   updateCurrency(args: UpdateCurrencyArgs, headers?: object, signal?: AbortSignal): Promise<UpdateCurrencyReturn>
-  listCurrencies(headers?: object, signal?: AbortSignal): Promise<ListCurrenciesReturn>
+  listCurrencies(args: ListCurrenciesArgs, headers?: object, signal?: AbortSignal): Promise<ListCurrenciesReturn>
```

Comment:
@AlexanderKolberg hmm, is this technically a breaking change?

Should we always generate `args: ListCurrenciesArgs` no matter if there are any input arguments?